auction-dutch/chaincode-go/smart-contract/auction.go

```diff
@@ -8,7 +8,8 @@ import (
 	"bytes"
 	"crypto/sha256"
 	"encoding/json"
-	"fmt"
+    "errors"
+    "fmt"
```

Comment:
This indentation looks to be spaces rather than tabs, as used by gofmt.

Reply:
Thanks, I have fixed it